src/components/network/equipment-table-editors.js

```diff
@@ -25,6 +25,21 @@ export const TapChangerSelector = ({
     defaultValue,
     ...props
 }) => {
+    const [arrayLength, setArrayLength] = useState(0);
+    const [value, setValue] = useState('');
```

Comment:
Why do we need a state for these ?
Is there really a need to handle the case where the props are like 5, then undefined, then 8, and we require that we still display 5 for the whole duration when the prop is undefined ?

Reply:
the problem here is when we have NaN as array length we get an error, so i added this state to check the value is not NaN before updating